compass/landice/mesh.py

```diff
@@ -102,3 +102,100 @@ def set_rectangular_geom_points_and_edges(xmin, xmax, ymin, ymax):
         dtype=jigsawpy.jigsaw_msh_t.EDGE2_t)
 
     return geom_points, geom_edges
+
+
+def set_cell_width(self, section, thk, vx=None, vy=None,
+                   distToEdge=None, distToGroundingLine=None):
+    """
+    Set cell widths based on settings in config file to pass to
+    :py:func:`mpas_tools.mesh.creation.build_mesh.build_planar_mesh()`.
+
+    Parameters
+    ----------
+    section : str
+        Section of the config file from which to read parameters
+    thk : numpy.ndarray
+        Ice thickness field from gridded dataset,
+        usually after trimming to flood fill mask
+    vx : numpy.ndarray
+        x-component of ice velocity from gridded dataset,
+        usually after trimming to flood fill mask. Can be set to None
+        if useSpeed == 'False' in config file.
+    vy : numpy.ndarray
+        y-component of ice velocity from gridded dataset,
+        usually after trimming to flood fill mask. Can be set to None
+        if useSpeed == 'False' in config file.
+    distToEdge : numpy.ndarray
```

Comment:
I'd prefer to move away from camel case in the framework as much as possible, but particularly in function arguments and config options (see below).
```suggestion
    dist_to_edge : numpy.ndarray, optional
```